server/bin/pbench-backup-tarballs.py

```diff
@@ -409,9 +406,9 @@ def backup_data(lb_obj, s3_obj, config, logger):
             )
             continue
 
-        resultname = os.path.basename(tar)
-        controller_path = os.path.dirname(tar)
-        controller = os.path.basename(controller_path)
+        resultname = Path(tar).name
```

Comment:
Let's create one `Path(tar)` object, and then use the `.name` and `.parent` methods instead of instantiating it twice.

Reply:
ok